packages/koa-shopify-webhooks/CHANGELOG.md

```diff
@@ -5,8 +5,16 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [1.1.1]
+
+### Fixed
+
+- Fixed a typo in the README
+
 ## [1.1.0]
 
+### Added
```

Comment:
If I'm understanding the 1.1.0 commit correctly, it can be `### Changed`, since it was just changing existing behaviour.